requirements.html

```diff
@@ -51,19 +55,40 @@ <h2 id="windows-version">Windows version</h2>
 <p>Parts of Git are implemented in shell script, and Git for Windows runs those scripts via <a href="https://msys2.github.io/">MSYS2</a>&#39;s POSIX emulation layer, which in turn is based on the <a href="https://cygwin.com">Cygwin POSIX emulation layer</a>. Seeing as Windows XP and Windows Server 2003 are years past their official end of life, the Cygwin project ended their Herculean efforts to support those Windows versions.</p>
 
 			</article>
+			</div>
 		</section>
-		<footer>
-			<div class="content">
-				<ul class="list-unstyled">
-					<li><a href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="http://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-				</ul>
-				<a href="index.html"><h1 class="gittext lowercase">Git for Windows</h1></a>
-				<div class="stud"></div>
+		<footer id="dk-footer" class="dk-footer">
+			<div class="footer">
+				<div class="container">
+					<div class="row">
+						<div class="col-md-6">
+							<span>Copyright © 2020 Johannes Schindelin and contributors. All rights reserved.</span>
```

Comment:
2021?

Reply:
> what advice Johannes and contributors have received

I haven't received any, apart from what you just said. I am in favor of dropping the parts you suggested to drop.

Reply:
> 2021?

@mjcheetham It's 2021 ALREADY???


Sorry. Had to do this.